golem-worker-service-base/src/api/register_api_definition_api.rs

```diff
@@ -239,15 +423,15 @@ pub struct MiddlewareData {
 #[oai(rename_all = "camelCase")]
 pub struct GatewayBindingWithTypeInfo {
```

Comment:
I think we should consider making this guy an enum when we can break backwards compatibility next time

Reply:
User facing response site, and I didn't want to bother too much about it.
Anything core logic that's done internally is type safe and its in enums as far as I believe.